readme-vars.yml

```diff
@@ -25,7 +25,10 @@ param_ports:
   - { external_port: "80", internal_port: "80", port_desc: "http gui" }
 param_usage_include_env: true
 param_env_vars:
-  - { env_var: "TZ", env_value: "Europe/London", desc: "Specify a timezone to use EG Europe/London"}
```

Comment:
Is the TZ var not needed?

Reply:
@aptalca I thought TZ and PUID and PGID were all included by default? Adding TZ back in since that doesn't seem to be the case.